package.json

```diff
@@ -435,9 +435,10 @@
     "@types/mocha": "^2.2.32",
     "@types/node": "^6.0.40",
     "@types/request-promise": "^4.1.41",
+    "@types/semver": "^5.5.0",
     "@types/sinon": "^5.0.5",
     "@types/strip-json-comments": "0.0.30",
-    "@types/semver": "^5.5.0",
+    "@types/tmp": "0.0.33",
```

Comment:
use ^ to match the latest minor release?

Reply:
^0.0.33 := >=0.0.33 <0.0.34   
So no need to use ^

Reply:
so we don't want to keep updated with minor version?  for example later it has 0.1.x?

Reply:
`0.0.5 -> 0.1.0` would be a little  different with `1.0.5 -> 1.1.0`. `0.0.5 -> 0.1.0` may contains breaking change. And `0.0.33` is auto generated by `npm install`, I think that is the reason